Extending/Property-Editors/tag-support.md

```diff
@@ -1,14 +1,14 @@
 ---
 versionFrom: 7.0.0
```

Comment:
If the version is only to V7 the versionfrom the tag should be removed and the the versionTo tag should say versionTo: 7.0.0 :-)